memcache/async_memcache.py

```diff
@@ -117,10 +117,10 @@ def __init__(
                 [AsyncConnection(addr, load_func=load_func, dump_func=dump_func)]
             )
 
-    def _get_connection(self, key) -> AsyncConnection:
+    def _get_connection(self, key: Union[str, bytes]) -> AsyncConnection:
         if isinstance(key, bytes):
             key = key.decode("utf-8")
-        return self._connections.get_node(key)
+        return self._connections.get_node(key)  # type: ignore[no-any-return]
```

Comment:
These ignores can be removed is the hashring library is updated with type annotations.